src/markdown_renderers.js

```diff
@@ -78,12 +80,15 @@ export const parseRepoName = (repoUrl) => {
  * Generates URLs for RAW content such as images
  *
  * @param {string} repoUrl
- * @param {string} branchName
+ * @param {string} gitBranchName
```

Comment:
Maybe when you are renaming this, could you also rename the `repoUrl`too to have this consistent?